### PR TITLE
enhc: making navbrand-name a link to homepage

### PR DESCRIPTION
**Pull request by kalosFestus**

### CODE REVIEW

File: practice.html [hunk 1]
```diff
@@ -13,12 +13,11 @@
...
             <div class="brand">
-                <h1 class="brand-name">Typing Test</h1>
+                <a href="./index.html" class="brand-name">Typing Test</a>
             </div>
             <div class="navigation">
                 <ul class="main-menu">
-                    <li class="nav-item"><a href="index.html" class="nav-item-link">Home</a></li>
-                    <li class="nav-item"><a href="practice.html" class="nav-item-link">Practice</a></li>
+                   <li class="nav-item"><a href="practice.html" class="nav-item-link">Practice</a></li>
                     <li class="nav-item"><a href="about.html" class="nav-item-link">About</a></li>
```
Comment (arpitghura) [inline practice.html:21]: Do proper alignment of `li`

File: profile.html [hunk 2]
```diff
@@ -15,12 +15,11 @@
...
         <div class="navbar">
             <div class="brand">
-                <h1 class="brand-name">Typing Test</h1>
+                <a href="./index.html" class="brand-name">Typing Test</a>
             </div>
             <div class="navigation">
                 <ul class="main-menu">
-                    <li class="nav-item"><a href="index.html" class="nav-item-link">Home</a></li>
-                    <li class="nav-item"><a href="practice.html" class="nav-item-link">Practice</a></li>
+                   <li class="nav-item"><a href="practice.html" class="nav-item-link">Practice</a></li>
```
Comment (arpitghura) [inline profile.html:22]: Do proper alignment of `li`

**Review by arpitghura — CHANGES_REQUESTED** [review]
Do proper alignment of all `li`.

File: practice.html (hunk 1 above)
Comment (kalosFestus) [reply practice.html:21]: all right, done, thanks

**Review by arpitghura — CHANGES_REQUESTED** [review]
**DONOT** Format all the files using any tool. Just properly align the li only.

**Comment by kalosFestus** [comment]
Okay, i will do that

**Review by arpitghura — APPROVED** [review]

**Pull request merged**